### ISSUE TITLE
Low accuracy value

### ISSUE THREAD

**Issue by adamchaabani** [LexicalStressDetection/lexical-stress-detection#2]
Hi, I am running the stress detection code.
First, I can't apply the alignment on the database after removing the stopwords there is an error and second and without removing the stopwords and after the train of the model I had an accuracy on the train 37% and on the test 36% which is very low compared to your project documentation, normally 89%..

**Comment by vivekkr12** [comment]
Since this model is essentially a binary classifier - detecting whether a particular phoneme has primary stress or not, the baseline accuracy should be ~50% without any training.

Did you follow the steps mentioned in README on how to train the model? Refer to this issues on some hints on re training the model. #1  

What error are you getting while running alignment?

**Comment by adamchaabani** [comment]
![Capture d’écran du 2022-08-15 19-36-57](https://user-images.githubusercontent.com/81832214/184697906-52306385-9265-4de2-9b59-2165d3354d76.png)

here is the error i got when i removed the stopwords and applied the alignment

**Comment by adamchaabani** [comment]
Yes sir I followed the readme file step by step except that I only worked with the librispeech database because I could not find where to download the OSCAAR database.
After the train test split stage I deleted the secondary stress file (2) and applied the train by varying the hyperparameters and still low accuracy. Could you help me please?

**Comment by vivekkr12** [comment]
Phoneme alignment doesn't need any stop words removal, all you need is a dictionary file. We skip feature generation of the stop words.

**Comment by adamchaabani** [comment]
the problem is that even without removing the stopwords it gave very low accuracy results compared to your project documentation, normally 89%. I don't know where the problem comes from

**Comment by vivekkr12** [comment]
What accuracy do you get while evaluating on test dataset without training? It should be ~50%.
